src/app/lorax/directives/modal-share.js

```diff
@@ -152,14 +153,21 @@ define(['angular', 'jquery'], function (angular, $) {
     };
 
     ModalShareController.prototype.shareFacebook = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/facebook/');
         window.open('https://www.facebook.com/sharer/sharer.php?u=' + issue.getShareUrl(), '_blank');
     };
 
     ModalShareController.prototype.shareTwitter = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/twitter/');
         window.open('http://twitter.com/share?text=' + issue.getTitle() + '&url=' +
             issue.getShareUrl() + '&hashtags=' + this._$scope.modalShare.content.hashtag, '_blank');
     };
 
+    ModalShareController.prototype.shareEmail = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/email/');
+        window.open('mailto:?subject=Shape of the Web - ' + issue.getTitle() + '&body=' + issue.getShareUrl());
```

Comment:
"Shape of the Web" needs to be localized here; perhaps pass the subject as an argument? Or just use the title as the subject like the old code?